src/lib/seo/structured-data.ts

```
@@ -0,0 +1,318 @@
+export interface StructuredDataConfig {
+  type: 'WebSite' | 'Organization' | 'Article' | 'FAQPage' | 'BreadcrumbList' | 'SoftwareApplication';
+  data: Record<string, any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)